.github/workflows/comprehensive-ci.yml

```
@@ -2,32 +1,103 @@
+        required: false
+        default: 'Pre-release validation'
+
+  # Trigger when specific label is added to PR
   pull_request:
-    branches: [ main, develop ]
-    paths:
-      - 'src/pipeline_v3/**'
-      - 'tests/**'
-      - '.github/workflows/pipeline_v3_ci.yml'
-      - 'pyproject.toml'
+    types: [ labeled ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)